appcircle_audit_permission_changes/1.0.0/component.yaml

```diff
@@ -0,0 +1,17 @@
+platform: iOS
+buildPlatform:
+displayName: Audit Permission Changes
+description: "Permission Differance Checker"
+webUrl: https://github.com/appcircleio/appcircle-ios-permission-check-component
```

Comment:
Second Review

**main.rb**

- [ ] `value.start_with?('$') ? ENV[value[1..-1]]`must be removed.
- [ ]  35, 36, 37, 39, 46, 47. line => If there is no relevant env variable in these lines, a Missing error should be returned. However, the "get_env_variable()" function does not return this error.
- [ ] "variable" param must be removed from `get_value_from_build_settings()` and `read_permissions_from_info_plist ()`functions and  `INFOPLIST_KEY_` must be const.
- [ ] 78. line => In this line `value.each do` must be remove and log must be add in 74. line
- [ ] 206. line => if condition must be removed because in every condition `permission_result..` name is same.
- [ ] 216. line => run_command_with_log() duplicated.
- [ ] 224. line => is `ac_output_dir()` function necessary?
- [ ] 248. line => `expand_exclude()` function is unnecessary.
- [ ] 312. - 315. lines => We have only one file for that we can remove excludes...() function and fix include...() function.